enos/provider/g5k.py

```diff
@@ -54,10 +54,7 @@ def init(self, conf, force_deploy=False):
         return (roles, network, (network_interface, external_interface))
 
     def destroy(self, env):
-        conf = load_config(env['config'],
-                           self.topology_to_resources,
-                           self.default_config())
-        provider_conf = conf['provider']
+        provider_conf = env['config']['provider']
```

Comment:
when destroying before uping : enotask is failing before reaching this part of the code.
Anyway, I agree it doesn't look right to access the member like this :)